nexus/src/external_api/console_api.rs

```diff
@@ -432,6 +433,7 @@ pub async fn login_local(
                 // browser expiration is mostly for convenience, as the API will
                 // reject requests with an expired session regardless
                 apictx.session_absolute_timeout(),
+                apictx.external_tls_enabled,
```

Comment:
I thought about having `session_cookie_header_value` take the whole `apictx`, but then the unit tests become less trivial, and it's nice for unit tests to be trivial.